src/handlers/simplehash.ts

```diff
@@ -154,6 +161,6 @@ export async function getNftsByWalletAddress(walletAddress: string) {
   return parseSimplehashNfts(rawResponseNfts).filter(
     (token: UniqueAsset) =>
       token.network !== Network.polygon ||
-      polygonAllowlist.includes(token.asset_contract?.address?.toLowerCase())
+      polygonAllowlist[token.asset_contract?.address?.toLowerCase() || '']
```

Comment:
this type can technically be undefined so we're falling back here to make ts happy